samples/cpp/speech_sample/main.cpp

```diff
@@ -1,523 +1,31 @@
 // Copyright (C) 2018-2021 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 //
-
-#include <gflags/gflags.h>
-#include <time.h>
-
+// clang-format off
 #include <chrono>
+#include <time.h>
 #include <fstream>
 #include <functional>
-#include <gna/gna_config.hpp>
-#include <inference_engine.hpp>
 #include <iomanip>
 #include <iostream>
 #include <limits>
 #include <map>
 #include <memory>
 #include <random>
-#include <samples/args_helper.hpp>
-#include <samples/common.hpp>
-#include <samples/slog.hpp>
 #include <string>
 #include <thread>
 #include <utility>
 #include <vector>
-
+#include <openvino/openvino.hpp>
+#include <gna/gna_config.hpp>
 #include "fileutils.hpp"
 #include "speech_sample.hpp"
+#include "utils.hpp"
+#include <samples/args_helper.hpp>
```

Comment:
Please finish reorder:  add empty lines between sections(compiler libs header, samples common headers,  particular sample headers) and need reorder these lines:
```
#include <openvino/openvino.hpp>
#include <gna/gna_config.hpp>
#include "fileutils.hpp"
#include "speech_sample.hpp"
#include "utils.hpp"
#include <samples/args_helper.hpp>
#include <samples/slog.hpp>
```

You can see good example about reorder in classification_sample_async/main.cpp

Reply:
Finished reorder

Reply:
Was done